lib/response.js

```diff
@@ -108,6 +108,14 @@ function generatePayload (response) {
   res.body = res.payload
   res.trailers = response._lightMyRequest.trailers
 
+  // Prepare payload parsers
+  res.json = function parseJsonPayload () {
+    if (this.headers['content-type'] !== 'application/json') {
```

Comment:
You should use `indexOf` instead. The most common form is `application/json; charset=utf-8` because... some browser compatibility issue :/.

Reply:
Done